data/scripts.js

```diff
@@ -1423,7 +1423,7 @@ exports.BattleScripts = {
 			item = 'Sitrus Berry';
 		} else if (template.species === 'Cubone' || template.species === 'Marowak') {
 			item = 'Thick Club';
-		} else if (template.species === 'Pikachu') {
+		} else if (template.species === 'Pikachu' || template.baseSpecies === 'Pikachu') {
```

Comment:
I didn't realize that `Tools.getTemplate` set the `baseSpecies` if it wasn't in pokedex.js, so I thought it might be undefined for normal Pikachu. It's fixed now.